teacherchannel/database.py

```diff
@@ -17,7 +17,7 @@ class TeacherChannel(database.base):
     master_id = Column(BigInteger, unique=True)
     slave_id = Column(BigInteger, primary_key=True, autoincrement=False)
     teachers = relationship(
-        "Teacher", cascade="all, delete-orphan", back_populates="teacherchannel"
+        lambda: Teacher, cascade="all, delete-orphan", back_populates="teacherchannel"
     )
 
     @staticmethod
```

Comment:
I'd like to see how would you cut the whole module into smaller PR's. It might be possible to make School and Review functions as two PR's, but since it's interconnected and it was constantly reworked during the development, it was not possible.